src/contracts/minter/libs/LibMint.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.14;
+
+// solhint-disable-next-line
+import {SafeERC20Upgradeable, IERC20Upgradeable} from "@openzeppelin/contracts-upgradeable/token/ERC20/utils/SafeERC20Upgradeable.sol";
+
+import {Arrays} from "../../libs/Arrays.sol";
+import {MinterEvent} from "../../libs/Events.sol";
+import {FixedPoint} from "../../libs/FixedPoint.sol";
+import {Math} from "../../libs/Math.sol";
+
+import {LibCalc} from "./LibCalculation.sol";
+import {KrAsset} from "../MinterTypes.sol";
+import {MinterState} from "../MinterState.sol";
+
+library LibMint {
```

Comment:
I think we could call this `LibFees` and move all fee stuff here? So close/open fees at this stage

Reply:
I agree, makes more sense to colocate the functions. We could combine them as the only difference is line 43 `krAsset.openFee` vs `krAsset.closeFee` and then line 69 `OpenFeePaid` vs `CloseFeePaid` and use Fee.Open/Fee.Close enum as a parameter...but it's not as clean, let's discuss on call today